eodatasets3/tesp_ard_metadata_only.py

```diff
@@ -30,6 +30,9 @@ def package_non_standard(outdir, granule):
 
     outdir = Path(outdir)
     out_fname = Path(str(granule.wagl_hdf5).replace("wagl.h5", "yaml"))
+    out_fname_bool = Path(str(granule.wagl_hdf5).replace("wagl.h5", "bool.wagl.h5"))
```

Comment:
Instead of "bool.wagl.h5" use something that provides a little more context such as:

"converted-datasets.h5"